features/alt-text-in-content.yml

```diff
@@ -0,0 +1,6 @@
+name: Alt text for generated content
```

Comment:
Sorry, I forgot to also ask to match the ID to the new name, whatever you picked.

So this file probably needs to become something like `alt-text-generated-content.yml`

Reply:
Done!